src/iuse.cpp

```diff
@@ -1248,7 +1248,7 @@ int iuse::plantblech(player *p, item *it, bool, const tripoint &pos)
         //reverses the harmful values of drinking fertilizer
         p->mod_hunger(p->nutrition_for(food) * multiplier);
         p->thirst -= food->quench * multiplier;
-        p->mod_healthy_mod(food->healthy * multiplier);
+        p->mod_healthy_mod(food->healthy * multiplier, food->healthy * multiplier);
```

Comment:
Some modifier to the cap would be useful here. Or maybe a hardcoded value (200 would be fine).
Health values of food are really low.